### PR TITLE
Added more projectile sources and the EXMODPID projectile mutator

### PR DESCRIPTION
**Pull request by OlvynChuru**

### CODE REVIEW

File: EEex/copy/EEex_Pro.lua [hunk 1]
```diff
@@ -44,6 +54,16 @@ EEex_DecodeProjectileSources = {
 	[EEex_Label("CGameAIBase::ForceSpell()_DecodeProjectile")      + 0x5] = EEex_ProjectileHookSource.FORCE_SPELL,
 	[EEex_Label("CGameAIBase::ForceSpellPoint()_DecodeProjectile") + 0x5] = EEex_ProjectileHookSource.FORCE_SPELL_POINT,
 	[EEex_Label("CGameSprite::UpdateAOE()_DecodeProjectile")       + 0x5] = EEex_ProjectileHookSource.UPDATE_AOE,
+	[5443518]                                                             = EEex_ProjectileHookSource.FORCE_SPELL_OPCODE,
```
Comment (Bubb13) [inline EEex/copy/EEex_Pro.lua:57]: These sources need to defined using patterns, so they work across all games / game versions.

Comment (OlvynChuru) [reply EEex/copy/EEex_Pro.lua:57]: I was using the latest version of BG:EE with SOD. I notice you added 0x5 to the source addresses. The ones I found likely have that extra +0x5 as well. When I subtract 0x5 from the addresses I found, I get these:

5443513
5446454
6093771
6109943
6120653
6135848
7518841
7520729
7529918
7532522

Do these align with any valid decode sources in BG:EE?

Comment (Bubb13) [reply EEex/copy/EEex_Pro.lua:57]: These turn out to be addresses for the Beamdog / GOG(?) distributions of SoD. I forgot those stores use a slightly different binary – that explains why the addresses didn't line up for me. I'll get to work on the patterns.

Comment (OlvynChuru) [reply EEex/copy/EEex_Pro.lua:57]: How is it going on figuring out the patterns? Is there something holding you up?

Comment (Bubb13) [reply EEex/copy/EEex_Pro.lua:57]: Hi, sorry about the delay. The addresses, being from an executable with no symbols, were harder to track down.

In the bottom right of this screen you should see an "Allow edits from maintainers" option - if you enable that I'll push the pattern edits and merge. :)

Comment (OlvynChuru) [reply EEex/copy/EEex_Pro.lua:57]: "Allow edits from maintainers" is enabled. Go ahead!

**Review by Bubb13 — APPROVED** [review]

**Pull request merged**